tests/phpunit/tests/test-links-domain-sub-dir.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * @group links
+ * @group domain
+ * @group directory
+ */
+class Links_Domain_Sub_Dir_Test extends PLL_Domain_UnitTestCase {
```

Comment:
DRY. This shouldn't be needed if you use a provider.

Reply:
Good idea! Done in a5d7ac57.